modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterManagementGroupManager.java

```diff
@@ -589,7 +591,7 @@ private void onElectedAsLeader(long term) {
                         }))
                         .whenComplete((v, e) -> {
                             if (e != null) {
-                                if (unwrapCause(e) instanceof NodeStoppingException) {
+                                if (hasCause(e, NodeStoppingException.class) || hasCause(e, CancellationException.class)) {
```

Comment:
I read the javadoc, but still not sure, given that there's a disjunction here. Can it be replaced with the following check?
```
hasCause(e, NodeStoppingException.class, CancellationException.class)
```